lib/src/input_validation/validate_function.dart

```diff
@@ -1,19 +1,27 @@
 import 'package:advanced_calculation/src/input_validation/error_state.dart';
 import 'package:advanced_calculation/src/input_validation/pattern.dart';
+import 'package:advanced_calculation/src/parse/expression_parser.dart';
+import 'package:advanced_calculation/src/translator/translate_pattern.dart';
 
 import 'start_state.dart';
 import 'state.dart';
 
 class ValidateFunction {
+  ExpressionParser parser = ExpressionParser();
+
   static final List<String> validFunctions = ["ln","log","sin","cos","tan", "abs", "csc","sec", "cot", "sqrt", "sinh", "cosh", "tanh",
     "asin", "acos", "atan", "acsc", "asec", "acot", "csch", "sech", "coth", "ceil","asinh", "acosh", "atanh", "acsch", "asech",
-    "acoth", "floor", "round", "trunc", "fract"];
+    "acoth", "floor", "round", "trunc", "fract", "√"];
   static final List<String> multiParamFunctions = ["max", "min", "gcd", "lcm"];
-
+  static final List<String> operands = ['*','/','-','+','(',')','^',',','²'];
```

Comment:
Operators, not operands. And this should probably be replaced with `Pattern.validAllOperators`. It's missing ) though for some reason

Reply:
and ² isn't needed because it is sanitized.

Reply:
correcting both

Reply:
need to add '=' in operand list

Reply:
this parsing doesnt append the = onto the end anymore as the user has no way of typing it